bitcoin-design/learning-bitcoin-and-design/fedimint.md

```diff
@@ -452,152 +445,155 @@ But basically you convince them, hey, I'm me and you have my shard of my seed an
-As you said, there's already some mechanism in there for that where I want to be able to through a contact be able to request a payment to someone else who I could identify within the Mint.
+Like the question isn't so much like like the capacity needed to storage capacity needed on the device but more like you know if I'm migrating to a different device I want to have like some metadata about my transactions you know and that's something that Bitcoin applications like really feel on or I want at least for my configurations to be the same and then we could extend that even further with some kind of like messaging system.
+As you said, there's already some mechanism in there for that where I want to be able to through a contact be able to request a payment to someone else who I could identify within the mint.
 
-Speaker 2: 00:41:59
+Justin Moon: 00:41:59
 
-Yeah, no, these are these are all great ideas.
+Yeah, these are all great ideas.
 I mean, these are totally unexplored at this point, but in order to get this in the wild, we'll need solutions for these things.
-So yeah, these are things that we'll have, apps being built on FedEvent we'll have to solve kind of similar to any other system.
+So yeah, these are things that we'll have, apps being built on FediMint we'll have to solve kind of similar to any other system.
 I mean, there could be interesting ways to have the Federation servers be able to relay messages, being able to store metadata.
 It's an interesting option.
-It's like you can kind of think of the servers as like your own private cloud in a sense as long as you can you know there might be some system in order to like request that you can do that but it's it's the federations are like a private cloud you just can't store too much in there right Because you have to have redundant copies and it's a bit of a tragedy of the commons, right?
+It's like you can kind of think of the servers as like your own private cloud in a sense as long as you can you know there might be some system in order to like request that you can do that but it's it's the federations are like a private cloud you just can't store too much in there, right?
+Because you have to have redundant copies and it's a bit of a tragedy of the commons, right?
 You don't want to be uploading videos, but that is a new constraint that you could design stuff around.
 But yeah, no plans for that so far.
 
-Speaker 5: 00:42:58
+Triston: 00:42:58
 
 All right.
 Thank you.
 
-Speaker 2: 00:43:00
+Justin Moon: 00:43:00
 
 It is a great observation though.
 Bitcoin walls do really suck it.
 
 ## Checking the status of the mint or proof-of-funds
 
-Speaker 4: 00:43:04
+You set a new one up it's like there's nothing in here you know.
```

Comment:
Same comment for this line as before. It makes sense to move it in the previous chapter